packages/styled/types/base.d.ts

```diff
@@ -68,7 +68,9 @@ export interface CreateStyledComponent<
 
   (
     template: TemplateStringsArray,
-    ...styles: Array<Interpolation<ComponentProps & SpecificComponentProps>>
+    ...styles: Array<
```

Comment:
could you add a type test for this?

Reply:
I will try, still need to figure how ^^ I was trying with local project to see if everything was following 😉

Reply:
I was hoping to have some tests failing in the CI to help me 😅 but everything is green 😕

Reply:
Should be fairly simple to add stuff here:
https://github.com/emotion-js/emotion/blob/31e610f2385d5a3dfd532b31f743e5f6b9fee43b/packages/styled/types/tests-base.tsx

Reply:
For sure, I just need to see how dtslint is working (my projects are usually directly in typescript 😅 ) Do you have a wait to execute `yarn test:typescript` on one file only or at least one project (and even better with a watch mode? 😇)